internal/display/roles.go

```diff
@@ -59,12 +58,11 @@ func (r *Renderer) RoleList(roles []*management.Role) {
 }
 
 func (r *Renderer) UserRoleList(roles []*management.Role) {
-	resource := "roles assigned to user"
+	resource := "user roles"
 	r.Heading(fmt.Sprintf("%s (%d)", resource, len(roles)))
 
 	if len(roles) == 0 {
-		r.Output("No roles assigned to user.\n\n")
-		r.Infof("Use 'auth0 users roles assign' to assign roles to a user.")
+		r.EmptyState(resource, "Use 'auth0 users roles assign' to assign roles to a user.")
```

Comment:
```suggestion
		r.EmptyState(resource, "Use 'auth0 users roles assign' to assign roles to a user")
```

Sometimes we print the hint with a period and sometimes not. I'm fine either way but let's stay consistent.